react-basic-practice/src/themes/index.ts

```diff
@@ -1,8 +1,12 @@
 import { extendTheme } from '@chakra-ui/react'
 import { buttonTheme } from './components/button'
 import { foundations } from './foundations'
+import { inputTheme } from './components/input'
```

Comment:
create index.ts file contains all components. After that, we just need import from `./component` and we can get all component with only one import